### ISSUE TITLE
All MVC4 downloads missing!

### ISSUE THREAD

**Issue by stevendc** [aspnet/Docs#5539]
## General

I can't get a download to the 2014 Web Camps training kit or the .NET 4 MusicStore solution. A boatload of people are not yet on .NET 5/.NET Core, so those downloads should still be available for training.

My main goal is training a team (and brushing up myself) on the proper patters for using dependency injection in a .NET 4 MVC app (also WebAPI). There are some big differences between .NET Core's DI and earlier versions, so I don't really want to train everyone on .NET Core at this time.

**Comment by guardrex** [comment]
Is it this :point_right: https://www.microsoft.com/en-us/download/details.aspx?id=36393

**Comment by stevendc** [comment]
It is indeed, but it results in essentially a 404 from Web Platform Installer. Any other direct download or alternative/ magnet ends in 404 as well.

![image](https://user-images.githubusercontent.com/24526800/36606261-329346ce-1891-11e8-815c-14389abdaceb.png)

**Comment by guardrex** [comment]
Ah ... `@jongalloway` may have the answer. See his comment at the end of this :point_right: https://forums.iis.net/t/1228981.aspx

**Comment by guardrex** [comment]
Curious: @terrimorton ... Is the page at https://www.microsoft.com/en-us/download/details.aspx?id=36393 in your ballpark? If so, can the page provide the content of Jon's note and have its download link disabled?

**Comment by stevendc** [comment]
I ran across that one in my initial Googling. The problem is the 2016 release is based on .NET Core, and that is significantly different from the way .NET 4/4.5 handles DI. I didn't even see a DI section in the 2016, which is my main interest in the .NET 4 Web Camps download. 

The articles and hands-on stuff is all still online, it's one of the top results when searching for DI training in MVC. I feel like the downloads should still be available too :(

**Comment by guardrex** [comment]
Does that include the 2015 release?

https://github.com/Microsoft-Web/WebCampTrainingKit/releases/tag/v2015.10.13b

**Comment by guardrex** [comment]
... and I see a bad link in that set of topics to: https://www.microsoft.com/download/29843 that 404s.

Also, the problem with using the link `http://aka.ms/webcamps-training-kit` is that it goes directly to the 2016 release. Those topics don't say how to reach the general releases page.

I'll take this on and ping Jon on the PR. @stevendc, let me know if that 2015 link get the right kit, and thanks for reporting this problem.

**Comment by stevendc** [comment]
The training I'm after is here, just so you know my hopeful goal:
https://docs.microsoft.com/en-us/aspnet/mvc/overview/older-versions/hands-on-labs/aspnet-mvc-4-dependency-injection

I'm reviewing the download above for 2015 release. It seems like the correct .NET version, but may not contain the specific DI training I was hunting. Either way thanks for the collection of fast responses, I'll continue to monitor this issue so any additional help is appreciated!

**Comment by guardrex** [comment]
@stevendc Yeah ... and I see that bad "29843" link there, too. Does the link down in the comments reach the correct project? https://github.com/Microsoft-Web/HOL-MVC4DependencyInjection/tree/master/Source

... and, yes, if you can let me know if that 2015 downloadable includes that MVC4 DI project, that would be great. Otherwise, np ... I'll download it and look.

ping @jongalloway 

TL;DR The Web Camp Training Kit link that replaces the Microsoft Download Center link, `http://aka.ms/webcamps-training-kit` redirects to `https://github.com/Microsoft-Web/WebCampTrainingKit/releases/tag/v2016.02.09`, so the reader doesn't immediately see that the 2015 release is still available at `https://github.com/Microsoft-Web/WebCampTrainingKit/releases/`. Do you have a preference for the docs between ...
* Leave the current link; but at the bottom of the topics, link out with the general releases that shows the 2016 **and** 2015 kits, **--OR--**
* Change the current link to point to the general releases page that shows both kits.

Also note that several topics link to https://www.microsoft.com/download/29843, and they should get the same update depending on which way you want to go. :point_up:

**Comment by stevendc** [comment]
@guardrex Yes, that's the correct solution located at https://github.com/Microsoft-Web/HOL-MVC4DependencyInjection/tree/master/Source

Thanks, from me and on behalf of the 6 devs who will be trained on DI with this exercise!!

**Comment by guardrex** [comment]
Np ... my pleasure. As soon as I hear back from Jon, I'll get those links updated.

**Comment by jongalloway** [comment]
I've updated https://aka.ms/webcamps-training-kit to point to the general releases list. Also, I think it makes sense to update notes at the end of the docs to link out to specific releases.

**Comment by guardrex** [comment]
Thanks @jongalloway ... then I'll just get the bad 404 link out at  https://www.microsoft.com/download/29843.

I'll ping you to review the PR. Thanks again! :rocket:

**Comment by guardrex** [comment]
@jongalloway 

> I think it makes sense to update notes at the end of the docs to link out to specific releases.

There is one thing that concerns me about doing that. We're going to be behind the 🎱 for future releases of the kit. I mean unless you (or someone) remembers to open issues going forward, we'll probably miss adding links for future training kits. Therefore, I think it best if we just update the broken link (`https://www.microsoft.com/download/29843`) to point to `https://aka.ms/webcamps-training-kit`.

@Rick-Anderson @scottaddie Thoughts?
